packages/react-core/src/components/Text/__tests__/TextList.test.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { TextList } from '../TextList';
+
+test('Renders without children', () => {
+  render(
+    <div data-testid="test-list">
+      <TextList />
+    </div>
+  );
+  expect(screen.getByTestId('test-list').firstChild).toBeVisible();
+});
+
+test('Renders children', () => {
+  render(<TextList>Test</TextList>);
+  expect(screen.getByText('Test')).toBeVisible();
+});
+
```

Comment:
If you add a test which asserts that an element has a class name of `''` and matches exactly would that work to test that it doesn't have any class names? I don't think we've ran into this question of how we want to test this behavior of components which have no classes by default yet, but I _think_ we may want to do that.

Reply:
Instead of asserting exactly that the element has a class name of `''`, I tested that there is no class name by using the follow matcher `not.toHaveClass()`. The `toHaveClass` documentation suggested this format when asserting whether there is no class name.